src/common/options.cc

```diff
@@ -7698,6 +7698,11 @@ std::vector<Option> get_mds_options() {
     .set_default(45)
     .set_description("timeout in seconds to wait for clients to reconnect during MDS reconnect recovery state"),
 
+    Option("mds_deny_all_reconnect", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(false)
+    .set_flag(Option::FLAG_RUNTIME)
+    .set_description("all reconnect req would be denied. Enabling this could load less inode before active, then shorten reboot time."),
```

Comment:
The convention is that `.set_description` should be a partial sentence without a terminating period (or other mark). Such that it could read: "[This config option is the] ... [.]". For example, the one below would read as:

> This config option is the **time in seconds between upkeep tasks**.

So I would recommend you use something like: "flag to deny all client reconnects during failover"

You can add a longer description via the method `.set_long_description`.

Reply:
> The convention is that `.set_description` should be a partial sentence without a terminating period (or other mark). Such that it could read: "[This config option is the] ... [.]". For example, the one below would read as:
> 
> > This config option is the **time in seconds between upkeep tasks**.
> 
> So I would recommend you use something like: "flag to deny all client reconnects during failover"
> 
> You can add a longer description via the method `.set_long_description`.

Thanks, changed

Reply:
You still need the short description (`.set_description`). Suggest you use the one I suggested.

Reply:
> You still need the short description (`.set_description`). Suggest you use the one I suggested.

ok changed